include/ck/wrapper/tensor.hpp

```diff
@@ -218,9 +242,14 @@ struct Tensor
         return layout_;
     }
 
-    // Getter for new sliced tensor
+    /**
+     * \brief Getter for new sliced tensor
+     *
+     * \param idx Passed indexes for slice/freeze.
```

Comment:
Please give an example of how one can define those indexes with slices and or freezes.